File: .NET Core/Exceptions/PostponeExecutionBackendException.cs
using com.contidio.sdk.proto;
using System.Net;

namespace Contidio.Sdk.Exceptions
{
    public class PostponeExecutionBackendException : BackendException
    {
        public PostponeExecutionBackendException()
            : base(BackendErrorCode.POSTPONE_EXECUTION, "Please try later", "PostponeExecutionBackendException", HttpStatusCode.NotAcceptable)
        {
        }
    }
}

File: Java/src/main/java/com/contidio/sdk/exceptions/PostponeExecutionBackendException.java
package com.contidio.sdk.exceptions;

import org.apache.commons.httpclient.HttpStatus;

import com.contidio.sdk.proto.MetadataProtocol.BackendErrorCode;

@SuppressWarnings("serial")
public class PostponeExecutionBackendException extends BackendException {

	public PostponeExecutionBackendException() {
		super(BackendErrorCode.POSTPONE_EXECUTION, "Please try later", "PostponeExecutionBackendException", 
				HttpStatus.SC_NOT_ACCEPTABLE);
	}

}
